Gemfile

```diff
@@ -30,7 +30,7 @@ gem "oa-openid", :require => "omniauth/openid"
 gem 'activerecord-session_store'
 gem 'actionpack-action_caching'
 gem 'rails-observers'
-
+gem "cancancan"
```

Comment:
@Alwahsh : You are correct. As a best practice we should specify gem versions on gemfile to avoid unexpected breakdowns while updating the bundler. I am working on this. thanks for the comment :)